### ISSUE TITLE
File won't minify

### ISSUE THREAD

**Issue by TomReddish-MVC** [mjvotaw/Coda-2-JS-Minify#5]
Had the plugin pointed to one of my js files. I removed the file from the plugin so that I could make a bunch of changes and then wanted to add the file back to minify when i was done. However when I added the file back, the plugin does not recognize any changes and does not create a minify file when I save the file. It was working before but seems like removing the file and adding it back causes an issue. Also I have show notification when error occurs minifying and nothing shows up as well when I save the file.

**Comment by mjvotaw** [comment]
Sorry to hear you're having problems with this plugin. I frankly haven't found the time to work on this plugin in quite a while, and I don't know when I'm going to get to this. Just wanted to warn you.

**Comment by jayenne** [comment]
yup. it used to work perfectly but somehow the plugin vanished and when i reinstalled it it wouldn't create the minified file (or save one I'd prepared earlier)

I suspect maybe a Coda update broke it? my current version is 2.6 (build 33594)
